roles/aws_cloudwatch_agent/templates/config.json.j2

```diff
@@ -37,6 +37,9 @@
 		}
 	},
 	"metrics": {
+{% if aws_cloudwatch_agent.metrics_namespace is defined and aws_cloudwatch_agent.metrics_namespace|length %}
+    "namespace": "MyCustomNamespace",
```

Comment:
Shouldn't this be `{{ aws_cloudwatch_agent.metrics_namespace }}`?

Reply:
it should, stupid merge conflict